index.html

```diff
@@ -0,0 +1,216 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=Edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script src="https://kit.fontawesome.com/3b9cd09213.js" crossorgin="anonymous"></script>
+    <link rel="preconnect" href="https:// fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link href="https://fonts.googleapis.com/css2?family=poppins:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css" <link rel="stylesheet"
+        href="https://fonts.googleleapis.com/css2?family=Material+Symbols+Rounded:ops,wght">
+    <title>MyPorfolio</title>
+</head>
+
+<body>
+    <div class="wraper">
+        <header>
+            <p>My Logo</p>
+            <nav><i class="fa-solid fa-bars"></i></nav>
+        </header>
+        <section class="headline">
+            <h1>Hello,I'm Khanya gald to see you! </h1>
+            <p class="details">
+                I'm a software developer! I can help you build a product, feature or
+                website Look through some of my work and experience! If you like what
+                you see and have a project you need coded, don't hesitate to contact me.
+            </p>
+            <h4>LET'S CONNECT</h4>
+            <div class="social">
+                <ul>
+                    <li><a href="twitter.com/khanya_mvoko"><i class="fa-brands fa-square-twitter"></i></a></li>
+                    <li><a href="linkedin.com/khanya-mvoko"><i class="fa-brands fa-linkedin"></i></a></li>
+                    <li><a href="github.com/Hotdog39duty"><i class="fa-brands fa-square-github"></i></a></li>
+                    <li><a href="#"><i class="fa-brands fa-angellist"></i></a></li>
+                </ul>
+            </div>
+        </section>
+    </div>
```

Comment:
- [ ] [OPTIONAL] As the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md#project-requirements) require, you should make sure that the Minimum desktop screen size should be `768px`. Kindly put this into your consideration and implement it.